### PR TITLE
[tests-only] Add makefile for UI test formatting

### PR DESCRIPTION
**Pull request by SwikritiT**
This PR adds Makefile for UI test formatting 
fixes https://github.com/owncloud/client/issues/8642

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#    - black
+#
+# Installing black can be done via https://black.readthedocs.io/en/stable/getting_started.html
+# configuration for black can be found inside pyproject.toml file under [tools.black] section.
+#
+# Usage:
+#
+#   make - auto format all the python files inside test/gui
+#   make check - check which files will be reformatted
+#	make diff - see what changes will be done
```
Comment (Talank) [inline Makefile:13]: Why is this line of comment has more space than other before the text? 🤔

**Review by Talank — COMMENTED** [review]
Since this is a Makefile for formatting python code for gui test, is it necessary to make this makefile in the root of the client? Can we not shift it in test/gui? Or maybe rename the commands such as  `make checkFormat`, `make diffFormat` 🤔

File: Makefile (hunk 1 above)
Comment (SwikritiT) [reply Makefile]: probably due to ide , fixed it

**Review by swoichha — COMMENTED** [review]
LGTM

**Review by individual-it — COMMENTED** [review]
LGTM, but I would love to know if @TheOneRing is happy with that extra Makefile

**Comment by TheOneRing** [comment]
I'm relative ok with it as its not in any src dir but it does not do what I would expect from a make file 😅 
As it is possible to pass files to make (-f) maybe use a more fitting name.
In the end, is there anything gained in introducing another scripting language to  a mostly python code base?
Why not just a format-code.py

**Comment by individual-it** [comment]
the idea was just to make it easier to remember what to type in that specific repo, in other repos all the test related commands, including source quality checks are in some `make` command. So I thought it would be easier to type `make` and see what it can than to go and search the docs for what the command is for that particular repo

**Comment by TheOneRing** [comment]
Then go for make

**Review by individual-it — APPROVED** [review]

**Pull request merged**